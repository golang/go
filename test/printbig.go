// run

// Copyright 2009 The Go Authors. All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

// Test that big numbers work as constants and print can print them.

package main

func main() {
	print(-(1<<63), "\n")
	print((1<<63)-1, "\n")
}
