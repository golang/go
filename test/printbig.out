-9223372036854775808
9223372036854775807
